bins/ggipcd/src/services/authorizationagent/token_validator.c

```diff
@@ -28,33 +29,52 @@ GglError ggl_handle_token_validation(
         )) {
         GGL_LOGE(
             "Component %.*s does not have access to token verification IPC "
-            "command",
+            "command.",
             (int) info->component.len,
             info->component.data
         );
+
+        *ipc_error = (GglIpcError
+        ) { .error_code = GGL_IPC_ERR_UNAUTHORIZED_ERROR,
+            .message = GGL_STR(
+                "Component does not have access to token verification IPC "
+                "command."
+            ) };
+
+        return GGL_ERR_FAILURE;
     }
+
     GglObject *svcuid_obj;
     GglError ret = ggl_map_validate(
         args,
         GGL_MAP_SCHEMA({ GGL_STR("token"), true, GGL_TYPE_BUF, &svcuid_obj }, )
     );
     if (ret != GGL_ERR_OK) {
         GGL_LOGE("Received invalid paramters.");
```

Comment:
Nit: Any reason to keep the log message vague?

Reply:
I agree that this logging can be better and more informative.

We have an ongoing effort to improve logging across board for greengrass lite.

And I do not want to balloon this PR beyond what it currently is.